### PR TITLE
Switch to initiator/target

### PR DESCRIPTION
**Pull request by kdb424**
Wording change to go along with world trends on open source code. Simple wording change, no functionality changed at all.

### CODE REVIEW

File: docs/split_keyboards.md [hunk 1]
```diff
@@ -14,22 +14,22 @@ keyboard.uart_pin = board.SCL
 Useful config options:
 ```python
 keyboard.split_flip = True  # If your boards are identical but one is flipped, this option is for you 
-keyboard.split_offsets = [6, 6, 6, 6]  # This is the how many keys are on each column on the "Master" half
+keyboard.split_offsets = [6, 6, 6, 6]  # This is the how many keys are on each column on the target" half
```
Comment (dzervas) [inline docs/split_keyboards.md:17]: missed a "

**Review by dzervas — COMMENTED** [review]
Correct me if I'm wrong but didn't you change both "master" and "side" words to target? wouldn't that introduce confusion?

File: docs/split_keyboards.md (hunk 1 above)
Comment (kdb424) [reply docs/split_keyboards.md:17]: This is correct. The "target" is the master half. This terminology can be found in fibre networks.

Comment (kdb424) [reply docs/split_keyboards.md:17]: Don't know what I thought I was responding to. Fixing. Thanks.

**Pull request merged**